src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -874,3 +874,132 @@
     text: >
         az acr check-health --ignore-errors
 """
+
+helps['acr scope-map'] = """
+type: group
+short-summary: Manage scope access maps for Azure Container Registries.
+"""
+
+helps['acr scope-map create'] = """
+type: command
+short-summary: Create a scope map for an Azure Container Registry.
+examples:
+  - name: Create a scope map that allows write and contentRead operations for `hello-world` repository, and pull operations for `hello/world`.
+    text: >
+        az acr scope-map create -r MyRegistry -n MyScopeMap --add hello-world;write,contentRead hello/world;pull --description "Sample scope map."
```

Comment:
Is there a way for user to know what actions/operations they can add like `write`, `contentRead`, etc?